plugins/module_utils/foreman_helper.py

```diff
@@ -448,12 +449,15 @@ def _delete_entity(self, resource, current_entity, params):
         self._resource_action(resource, 'destroy', payload)
         return True, None
 
-    def _resource_action(self, resource, action, params):
+    def resource_action(self, *args, **kwargs):
```

Comment:
Why not renaming the method altogether?
And if we think we need the hidden version for the transition, we should make it the alias and deprecate it.

Reply:
mostly me being lazy and not motivated to do `sed -i …` :)

Reply:
Then you should consider this as "external motivation (tm)".

Reply:
I motivated my vim to do the change for me.